metrics/examples/prometheus/install/prometheus-rules.yaml

```diff
@@ -74,4 +121,53 @@ spec:
         severity: warning
       annotations:
         summary: 'Zookeeper is running out of free disk space'
-        description: 'There are only {{ $value }} bytes available at {{ $labels.persistentvolumeclaim }} PVC'
\ No newline at end of file
+        description: 'There are only {{ $value }} bytes available at {{ $labels.persistentvolumeclaim }} PVC'
+    - alert: ZookeeperContainerRestartedInTheLast5Minutes
+      expr: count(count_over_time(container_last_seen{container_name="zookeeper"}[5m])) > 2 * count(container_last_seen{container_name="zookeeper",kubernetes_pod_name=~".+-zookeeper-[0-9]+"})
+      for: 5m
+      labels:
+        severity: warning
+      annotations:
+        description: 'Some of the Zookeeper containers was restarted too many times in last 5 minutes. This alert can be ignored during scaling up the Zookeeper cluster.'
```

Comment:
In general these _Some ..._ messages seem to be terrible. We should improve the message here. Maybe @laidan6000 can help with this?

Reply:
Suggested message:

One or more Zookeeper containers were restarted too often within the last 5 minutes. You can ignore this alert when scaling up the Zookeeper cluster.

Reply:
(Is "one or more" accurate?)